server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -185,6 +185,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion INCLUDE_INDEX_MODE_IN_GET_DATA_STREAM = def(9_023_0_00);
     public static final TransportVersion MAX_OPERATION_SIZE_REJECTIONS_ADDED = def(9_024_0_00);
     public static final TransportVersion RETRY_ILM_ASYNC_ACTION_REQUIRE_ERROR = def(9_025_0_00);
+    public static final TransportVersion INFERENCE_CONTEXT = def(9_026_0_00);
```

Comment:
Just a reminder, if we do want to backport to 8.19 we'll need a `TransportVersion` for 8.x

for example: `COHERE_BIT_EMBEDDING_TYPE_SUPPORT_ADDED_BACKPORT_8_X`

We'll also need to change the `onAfter()` check. Here's an example:
https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/embeddings/CohereEmbeddingType.java#L131-L132

The code in 8.x will look different too (since the 9.x transport version won't exist): https://github.com/elastic/elasticsearch/blob/8.x/x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/embeddings/CohereEmbeddingType.java#L131

Reply:
Thanks for the explanation and the code examples.

Adjusted with [Add TransportVersion for 8_X](https://github.com/elastic/elasticsearch/pull/124025/commits/e5153a120126d7e9202486f304247e5f5262ed2e).

In the backport I would then need to replace `TransportVersions.INFERENCE_CONTEXT` with `TransportVersions.INFERENCE_CONTEXT_8_X`, right?